### ISSUE TITLE
Any inventory changes should be going through InventoryManager

### ISSUE THREAD

**Issue by MarcinSc** [MovingBlocks/Terasology#1693]
At the moment, a couple of places in the code (for example ItemSystem) modify the inventory directly, rather than use InventoryManager. As a result - some events that would normally be sent are not being sent.

**Comment by Cervator** [comment]
Did #1695 relate to this at all?

**Comment by MarcinSc** [comment]
Not really, it added one more method that might be required to eliminate all non-InventoryManager usages though.

**Comment by Cervator** [comment]
@Josharias do you think this is complete after you extracted inventory from the engine into Core? Plain `ItemSystem` doesn't even exist anymore, at least not by that exact name.

**Comment by Josharias** [comment]
Hrm.  I don't recall any places that do this from my extraction of inventory from the engine.  I'll do a double check next time I have Terasology open.

**Comment by skaldarnar** [comment]
Just so that we don't have to wait another year :smile: Did you have a look at this in the meantime @Josharias ?
